README.md

```diff
@@ -34,76 +34,57 @@ sed -i "s/02:FF:FF:FF:FF:FF/$mac_oui:FF:FF:FF/" kubemacpool.yaml
 kubectl apply -f ./kubemacpool.yaml
 ```
 
-### Opting-in to kubemacpool service
+### kubemacpool service
 
-Kubemacpool is set to allocate mac to [supported interfaces](#About) on pods/vms that reside only on opted-in namespaces. You can opt-in your namespace by adding the following labels:
-- `mutatepods.kubemacpool.io=allocate` - to opt in pods mac allocation in your namespace
-- `mutatevirtualmachines.kubemacpool.io=allocate` - to opt in vms mac allocation in your namespace
+Kubemacpool is set to allocate mac to [supported interfaces](#About) on pods/vms using two webhooks: mutatepods and mutatevirtualmachines.
+While in [testing manifests](config/test/kubemacpool.yaml) both webhooks are set to be in opt-out mode, [release manifests](config/release/kubemacpool.yaml) set mutatepods to opt-in and mutatevirtualmachines to opt-out.
+You can change mutatepods back to opt-out in [release manifests](config/release/kubemacpool.yaml) by changing the kustomization.yaml:
 
-#### How to enable/disable kubemacpool for a namespace
+```bash
+sed -i 's/mutatepods_opt_in_patch.yaml/mutatepods_opt_out_patch.yaml/' config/release/kustomization.yaml
+make generate-deploy
+```
```

Comment:
Below this, add a note (formatted as a markdown quote) saying that for production the default is X but on testing Y.